src/WebJobs.Script/ManagedDependencies/ManagedDependencyOptions.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Microsoft.Azure.WebJobs.Script.ManagedDependencies
+{
+    public class ManagedDependencyOptions
```

Comment:
Options should reflect the managed dependency structure in host.json similar to: https://github.com/Azure/azure-functions-host/blob/dev/src/WebJobs.Script/BindingExtensionBundle/ExtensionBundleOptions.cs